src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationService.kt

```diff
@@ -60,8 +61,25 @@ class NotificationService(
 
         allPrisons.forEach { prison ->
             saveNotifications(
-                    csrClient.getShiftTaskNotifications(prison.csrPlanUnit, prison.region)
-                            .distinct())
+                   csrClient.getShiftTaskNotifications(prison.csrPlanUnit, prison.region)
+                            .map {
+                                if(ShiftNotificationType.from(it.shiftType) == ShiftNotificationType.SHIFT || it.shiftType.equals("task",true)) {
```

Comment:
what's the difference between the `==` assertion and the `.equals` assertion here?

Reply:
the == is we're changing the type to an enum so we can we compare two of the same type so we're comparing instances.
the other one is the sql sends 'task' sometimes which I think is a bug/deployment issue that I don't want to address because legacy. so its a case insensitive value equals check. because 'task' isn't a valid shift type we can't change the type to an enum first and do an ==